lib/rucio/web/rest/flaskapi/v1/replicas.py

```diff
@@ -298,6 +298,9 @@ def post(self):
                            'site': None}
         client_location.update(param_get(parameters, 'client_location', default={}))
 
+        # making sure IP address is not overwritten
+        client_location['ip'] = client_ip
```

Comment:
There is a HTTP GET call as well for list_replicas in line 54. Needs to be done there too.

Reply:
that one isn't overwritten?

Reply:
No, you only did it for the POST call. I know it is quite confusing that there are two different ones, without going into to much details, I blame historic reasons 😄